nsqadmin/http.go

```diff
@@ -641,6 +655,14 @@ func (s *httpServer) counterHandler(w http.ResponseWriter, req *http.Request, ps
 		s.ctx.nsqadmin.logf(LOG_WARN, "%s", err)
 		messages = append(messages, pe.Error())
 	}
+
+	for _, producer := range producers {
+		// Assume it is an ipv6 address if a colon is in it
+		if strings.Contains(producer.BroadcastAddress, ":") {
+			producer.BroadcastAddress = fmt.Sprintf("[%s]", producer.BroadcastAddress)
```

Comment:
looks like this place is also inappropriate because this is going to be used for `GetNSQDStats()`, not for web UI links